ntbs-service/Helpers/NameFormattingHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Text.RegularExpressions;
+
+namespace ntbs_service.Helpers
+{
+    public static class NameFormattingHelper
+    {
+        private static readonly Regex DisplayNameRegex = new Regex(@"^\s*([^\(\s](\s*[^\(\s]+)*)\s*(\(.*)?$");
```

Comment:
I think this regex is sufficiently complicated that it has become very difficult to read/understand (and therefore to maintain). I think this would be much clearer if we use C# string operations. I think we can just:
- Split the string on the open bracket character
- Take the first group
- Trim it

Reply:
Yeah sure, it made perfect sense yesterday but after looking at it even a day later and I agree it's not easy to understand! Changed as you outlined in 0f7b168